agent/stats/engine_unix_test.go

```diff
@@ -0,0 +1,99 @@
+//+build unit
```

Comment:
why is the test unix specific? the "default" mode is for windows right

Reply:
default is for both linux, windows. but good catch, will add this mode to non os specific file.